### DISCUSSION TITLE
触摸屏上，web页面table表格无法进行手指滑动。其实是web界面所有能滑动的都不行了 鼠标滚轮ok

### DISCUSSION THREAD

**Discussion by TiankeTt** [CefView/QCefView#426] [Q&A]
触摸屏上，web页面table表格无法进行手指滑动。其实是web界面所有能滑动的都不行了 鼠标滚轮ok

**Comment by tishion** [comment]
OSR模式不支持触屏

**Comment by TiankeTt** [comment]
关闭这个模式好像会崩。我们同事反馈


---- 回复的原邮件 ----
| 发件人 | Sheen ***@***.***> |
| 日期 | 2024年08月08日 18:19 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [CefView/QCefView] 触摸屏上，web页面table表格无法进行手指滑动。其实是web界面所有能滑动的都不行了 鼠标滚轮ok (Discussion #426) |

OSR模式不支持触屏
